backend/user_service/app.js

```diff
@@ -7,7 +7,11 @@ const app = express();
 
 // Allow requests from 'https://question-service-1079323726684.asia-southeast1.run.app' (cloudrun question_service) with credentials
 app.use(cors({
-    origin: ['https://frontend-1079323726684.asia-southeast1.run.app', 'http://localhost:3000', 'http://localhost:5173'],
+    origin: ['https://frontend-1079323726684.asia-southeast1.run.app',
+      'http://localhost:3000',
+      'http://localhost:5173',
+      'https://question-service-1079323726684.asia-southeast1.run.app',
+      'http://localhost:5000'],
```

Comment:
Is this a typo? What does `http://localhost:5000` belong to?

Reply:
ah yea it was a typo my bad i'll go fix it :skull: